app/src/main/java/com/github/kulebin/myfoursquareapp/imageLoader/ImageLoader.java

```diff
@@ -21,8 +27,12 @@
         int getHeight();
     }
 
+    private static final String TAG = ImageLoader.class.getSimpleName();
     private static final int CHUNK_SIZE_POWER = 16; //chunk size = 2^x
+    private static final int MAX_THREAD_NUMBER = 3;
 
+    private final ExecutorService mExecutorService = Executors.newFixedThreadPool(MAX_THREAD_NUMBER);
+    private final Handler mHandler = new Handler(Looper.getMainLooper());
     private ISizeParams mSizeParamsCallback;
```

Comment:
Probably we can return byte[] from httpClient, or create converter instance  for every call with size